### ISSUE TITLE
[Bug] Cannot start dash.

### ISSUE THREAD

**Issue by SangeloDev** [MauriceNino/dashdot#193]
### Description of the bug

This is the error when trying to start dashdot:
```js
node:internal/fs/utils:345
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/home/yunohost.app/dashdot/version.json'
    at Object.openSync (node:fs:585:3)
    at readFileSync (node:fs:453:35)
    at /home/yunohost.app/dashdot/dist/apps/api/main.js:2:2650516
    at /home/yunohost.app/dashdot/dist/apps/api/main.js:2:2653387
    at Object.<anonymous> (/home/yunohost.app/dashdot/dist/apps/api/main.js:2:2653619)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/home/yunohost.app/dashdot/version.json'
}
```
And this is the command I use to launch dashdot:
```bash
export DASHDOT_PORT="3002" &&\
export DASHDOT_ENABLE_CPU_TEMPS='true' &&\
export DASHDOT_ALWAYS_SHOW_PERCENTAGES='true' &&\
export DASHDOT_WIDGET_LIST='os,cpu,storage,ram' &&\
sudo yarn start
```

### How to reproduce

1. Clone the repo and build it.
```
git clone https://github.com/MauriceNino/dashdot &&\
  cd dashdot &&\
  yarn &&\
  yarn build:prod
  ```
2. Try to start dashdot with above mentioned command.
3. Errors.

### Relevant log output

```shell
node:internal/fs/utils:345
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/home/yunohost.app/dashdot/version.json'
    at Object.openSync (node:fs:585:3)
    at readFileSync (node:fs:453:35)
    at /home/yunohost.app/dashdot/dist/apps/api/main.js:2:2650516
    at /home/yunohost.app/dashdot/dist/apps/api/main.js:2:2653387
    at Object.<anonymous> (/home/yunohost.app/dashdot/dist/apps/api/main.js:2:2653619)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/home/yunohost.app/dashdot/version.json'
}
```

### Info output of dashdot cli

```shell
INFO
=========
Yarn: 3.2.1
Node: v16.16.0
Dash: unknown

Cwd: /home/yunohost.app/dashdot
Hash: 015eab35801f45646399d9d3e958b792d763a5cd

In Docker: false
In Podman: false
In Docker (env):
```

### What browsers are you seeing the problem on?

Firefox

### Where is your instance running?

Linux Server

**Comment by MauriceNino** [comment]
Thank you for the elaborate description! This will be fixed in the next release.

**Comment by MauriceNino** [comment]
:tada: This issue has been resolved in version 4.3.1

Please check the [changelog](https://github.com/MauriceNino/dashdot/blob/main/.github/CHANGELOG.md) for more details.
